python/cuml/cuml/internals/base.pyx

```diff
@@ -658,41 +658,16 @@ class UniversalBase(Base):
     def gpu_to_cpu(self):
         # transfer attributes from GPU to CPU estimator
         for attr in self.get_attr_names():
-            # check presence of attribute
-            if hasattr(self, attr) or \
-               isinstance(getattr(type(self), attr, None), property):
-                # get the cuml attribute
-                if hasattr(self, attr):
-                    cu_attr = getattr(self, attr)
-                else:
-                    cu_attr = getattr(type(self), attr).fget(self)
-                # if the cuml attribute is a CumlArrayDescriptorMeta
-                if hasattr(cu_attr, 'get_input_value'):
-                    # extract the actual value from the
-                    # CumlArrayDescriptorMeta
-                    cu_attr_value = cu_attr.get_input_value()
-                    # check if descriptor is empty
-                    if cu_attr_value is not None:
-                        if cu_attr.input_type == 'cuml':
-                            # transform cumlArray to numpy and set it
-                            # as an attribute in the CPU estimator
-                            setattr(self._cpu_model, attr,
-                                    cu_attr_value.to_output('numpy'))
-                        else:
-                            # transfer all other types of attributes
-                            # directly
-                            setattr(self._cpu_model, attr,
-                                    cu_attr_value)
-                elif isinstance(cu_attr, CumlArray):
```

Comment:
By my reading, none of this branch could ever be executed. Will see if CI agrees with me.